examples/launch_height_sagemaker.py

```diff
@@ -59,6 +60,8 @@
     scheduler = RandomSearch(
         config_space, mode=mode, metric=metric, random_seed=random_seed
     )
+    if "AWS_DEFAULT_REGION" not in os.environ:
+        os.environ["AWS_DEFAULT_REGION"] = "us-west-2"
```

Comment:
Is that needed? I think it was working for me but perhaps it is needed in case you dont have a aws config defined?

Reply:
Exactly what you mention. I can remove it and add a comment for users to add `AWS_DEFAULT_REGION` to their environment if you think thats better.